src/proxy/http_context.rs

```diff
@@ -162,8 +162,22 @@ impl Context for HttpAuthThreescale {
             info!(self, "on_http_call_response: authorized {}", token_id);
             self.resume_http_request();
         } else {
+            let rejection_reason = match self
+                .get_http_call_response_headers()
+                .into_iter()
+                .find(|(key, _)| key.as_str() == "3scale-rejection-reason")
+            {
+                None => "None".to_string(),
+                Some((_, reason)) => reason,
+            };
+
             info!(self, "on_http_call_response: not authorized {}", token_id);
```

Comment:
what about moving this logging its own branch?

Reply:
Moved